neuro_extras/image.py

```diff
@@ -334,6 +334,9 @@ async def _build_image(
                     f"Use -F/--force-overwrite flag to enforce overwriting."
                 )
 
+        if preset is None:
+            preset = await _select_build_preset(client)
+
```

Comment:
We could pass this preset to `_select_build_preset` and check whether it satisfies minimal requirements. Print warning if no.